extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -150,11 +106,7 @@ class="mc-submit-param"
 						value="<?php echo esc_attr( $values['signupFieldValue'] ); ?>"
 					/>
 				<?php endif; ?>
-				<p>
-					<button type="submit" class="<?php echo esc_attr( $button_classes ); ?>" style="<?php echo esc_attr( $button_styles ); ?>">
-						<?php echo wp_kses_post( $values['submitButtonText'] ); ?>
-					</button>
-				</p>
+				<?php echo render_button( $attr, $content ); // phpcs:ignore ?>
```

Comment:
I'll leave it to JP folks to chime in, but I wonder if it would make more sense to only disable the broken PHPCS rule:
```php
// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
echo render_button( $attr, $content );
// phpcs:enable WordPress.Security.EscapeOutput.OutputNotEscaped
```

Reply:
I'd agree with that as it gives as a clear look at why and what we are ignoring 👍

Reply:
I believe you could even do it on one line:

```suggestion
				<?php echo render_button( $attr, $content ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```